test/validate/windows_validate.go

```diff
@@ -22,6 +22,7 @@ var (
 	hnsEndPointCmd                 = []string{"powershell", "-c", "Get-HnsEndpoint | ConvertTo-Json"}
 	hnsNetworkCmd                  = []string{"powershell", "-c", "Get-HnsNetwork | ConvertTo-Json"}
 	azureVnetCmd                   = []string{"powershell", "-c", "cat ../../k/azure-vnet.json"}
+	azureVnetStatelessCmd          = []string{"powershell", "-c", "cat ../../k/azure-cns/azure-endpoints.json"}
```

Comment:
This is not solely a stateless command. It is a for validating cns managed statefile.

Reply:
Will use CNS template on linux